src/h5web/explorer/EntityList.tsx

```diff
@@ -23,38 +24,17 @@ function EntityList(props: Props): ReactElement {
     <ul className={styles.group} role="group">
       {entities.map((entity) => {
         const { uid, name } = entity;
-        const isExpanded = expandedGroups.has(entity.uid);
+        const path = `${parentPath === '/' ? '' : parentPath}/${name}`;
```

Comment:
We construct the entity's path ourselves. This is the key to the whole thing, really, since at the start, the only path we know exists is `/`. We could require providers to include a path in a group's children's metadata, but this seems easier and more robust.